Makefile

```diff
@@ -90,7 +90,8 @@ test-failpoint:
 	@echo "[failpoint] array freelist test"
 	BBOLT_VERIFY=all TEST_FREELIST_TYPE=array go test -v ${TESTFLAGS} -timeout 30m ./tests/failpoint
 
-.PHONY: test-robustness # Running robustness tests requires root permission
+.PHONY: test-robustness # Running robustness tests requires root permission for now
+# TODO: Remove sudo once we fully migrate to the prow infrastructure
```

Comment:
Question on the TODO comment: won't removing sudo on prow break the robustness test, since it requires root permission?

Reply:
Prow jobs already run as the super user, and `sudo` is not installed. When we migrated etcd jobs, we had to remove `sudo` calls. (cc. @jmhbnz)